Cabal/Distribution/Simple/Compiler.hs

```diff
@@ -26,6 +26,7 @@ module Distribution.Simple.Compiler (
         Compiler(..),
         showCompilerId, showCompilerIdWithAbi,
         compilerFlavor, compilerVersion,
+        compilerCompatFlavor,
```

Comment:
@23Skidoo this one appears to be an API addition, so if it doesn't make it into Cabal-1.24.0.0 it could only make it into Cabal-1.24.1.0 . And that's the min-version `cabal-install-1.24.0.1` would need to depend on